keyboards/ogurec/readme.md

```diff
@@ -0,0 +1,18 @@
+# ogurec
+
+![ogurec](https://imgur.com/gallery/OFF1Vyp)
+
+ogurec is a reversible 12u x 3u ortholinear keyboard that offers left-sided and right-sided PM placement
+
+* Keyboard Maintainer: [H. Bond](https://github.com/drhigsby)
+* Hardware Supported: ogurec (https://github.com/drhigsby/ogurec)
+* Hardware Availability: open source 3DP case available here https://github.com/drhigsby (case creator: https://github.com/dpalka15)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make ogurec:default_right
+    make ogurec:default_left
```

Comment:
```suggestion
    make ogurec:default
```
This board needs a single `default` keymap.

Reply:
@fauxpark Hey homie, I don't have a keymap called "default." Can I have just: 

> make ogurec:default_right

Will that be okay?

Reply:
No, there are certain parts of QMK and its tooling that assume every keyboard has a `default` keymap, in particular https://qmk.fm/keyboards/.

Reply:
Okay, that sounds like a good idea. I will look into re-organizing it that way. Thank you!

Reply:
updates here: https://github.com/qmk/qmk_firmware/pull/13242/commits/f551b86424eb15f134f98661f8e1d791bb756470

Reply:
I didn't mean `left` and `right` keymaps, I meant splitting the keyboard folder itself into `ogurec/left` and `ogurec/right`, with their own set of layout macros and individual `default` keymaps.

Reply:
I made some significant changes to the repo to do just this, but am having a lot of issues with it now. Not sure what I did wrong.

Reply:
okay, everything is now compiling correctly after further testing